src/main/java/rife/bld/publish/PomBuilder.java

```diff
@@ -121,6 +124,19 @@ public String build() {
         return StringUtils.stripBlankLines(t.getContent());
     }
 
+    /**
+     * Generates a POM into the given file.
+     *
+     * @since 1.7.1
+     */
+    public static void generateInto(PublishInfo info, DependencyScopes dependencies, File file)
+            throws FileUtilsErrorException {
+        var pomBuilder = new PomBuilder();
```

Comment:
Maybe `var pom_builder = new PomBuilder().info(info).dependencies(dependencies);`

then remove the next two lines?

Reply:
Back to my original thoughts. Then you are also creating a couple additional objects. Yet it matches the builder pattern used throughout the code base.

Done. :wink: